paasta_tools/api/views/instance.py

```diff
@@ -726,10 +738,14 @@ def instance_status(request):
             )
         elif instance_type == "flink":
             status = flink_instance_status(instance_status, service, instance, verbose)
+            metadata = flink_instance_metadata(
+                instance_status, service, instance, verbose
+            )
+            instance_status["flink"] = {}
             if status is not None:
-                instance_status["flink"] = {"status": status}
-            else:
-                instance_status["flink"] = {}
+                instance_status["flink"]["status"] = status
```

Comment:
are you sure you want to leave this empty when status is None, rather than leaving the response as an empty dictionary?

Reply:
yes, that preserves the old behaviour, I think.